### PR TITLE
Add support for magnatune.com

### LINKED ISSUE #2063: magnatune
**Provide information about website**
A short description of the website, sample URLs, (demo) accounts for testing purposes. If (demo) accounts are not for public use, you can mail them to the project developers.

Curated Creative Commons music site.

No account needed for streaming from the site. For example: 
http://magnatune.com/artists/albums/mburaglia-plaintesdepsyche/

**Additional context**
Add any other context or screenshots about the request here.

### PR DESCRIPTION
**Pull request by inverse**
Resolves #2063

### CODE REVIEW

File: src/connectors/magnatune.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		return null;
+	}
+
+	const trackContent = playingImg.closest('td').nextElementSibling.textContent;
+	const matched = trackContent.match(trackRegex);
+	if (!matched) {
+		return null;
+	}
+
+	return matched[1].trim();
```
Comment (alexesprit) [inline src/connectors/magnatune.js:26]: Trim is redundant here.

File: src/connectors/magnatune.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+};
+
+Connector.getAlbum = () => {
+	const artistElement = document.querySelector(artistSelector);
+
+	return artistElement.closest('font').innerText.replace(artistElement.textContent, '').slice(1);
```
Comment (alexesprit) [inline src/connectors/magnatune.js:32]: `innerText` can be replaced with `textContent`.

File: src/connectors/magnatune.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+
```
Comment (alexesprit) [inline src/connectors/magnatune.js:2]: These empty lines should be removed.

**Review by alexesprit — CHANGES_REQUESTED** [review]

File: src/core/connectors.js [hunk 4]
```diff
@@ -1656,6 +1656,11 @@ const connectors = [{
 	matches: ['*://laut.fm/*'],
 	js: 'connectors/laut.fm.js',
 	id: 'laut.fm',
+}, {
+	label: 'Mmagnatune',
```
Comment (alexesprit) [inline src/core/connectors.js:1660]: There's a typo here.

File: src/connectors/magnatune.js (hunk 2 above)
Comment (inverse) [reply src/connectors/magnatune.js]: This always gets me!

**Comment by inverse** [comment]
@alexesprit done

**Review by alexesprit — APPROVED** [review]

**Pull request merged**